emspy/query/fltquery.py

```diff
@@ -155,9 +178,9 @@ def __translate_expr(self, expr_vec):
 			else:
 				if type(x) != list:
 					x = [x]
-				val_info = [{'type':'constant','value':v} for v in x]
+				val_info += [{'type':'constant','value':v} for v in x]
 
-		if fld_loc[1]:
+		if fld_loc[1] and len(expr_vec) == 3:
```

Comment:
Comment here?  Since the len(expr_vec) == 3 bit is new.

Reply:
See comment after d5c3a44, it seems like the code in this block

https://github.com/ge-flight-analytics/emspy/blob/d5c3a44d763c3b2de231e39540b58d4f0302f3a7/emspy/query/fltquery.py#L189-L194 

is unreachable